pages/instructorAssessmentGroups/instructorAssessmentGroups.ejs

```diff
@@ -0,0 +1,445 @@
+          <div class="col-sm bg-light py-4 border" align="center">
+            <button type="button" class="btn btn-primary text-nowrap" data-toggle="modal"
+              data-target="#autoAssessmentGroupsModal">
+              <i class="fas fa-robot" aria-hidden="true"></i> Auto
+            </button>
+            <div class="mt-2">
+              Automatically assign students to groups.
+            </div>
+          </div>
+          <div class="col-sm bg-light py-4 border" align="center">
+            <button type="button" class="btn btn-primary text-nowrap" data-toggle="modal"
+              data-target="#copyAssessmentGroupsModal">
+              <i class="fas fa-copy" aria-hidden="true"></i> Copy
+            </button>
+            <div class="mt-2">
+              Copy the group assignments from another assessment.
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="table-responsive">
+        <table id="usersTable" class="table table-sm table-hover tablesorter">
+          <thead>
+            <tr>
+              <th>Name</th>
+              <th>Size</th>
+              <th class="text-center">Group Members (UIDs)</th>
+              <th class="sorter-false"></th>
+            </tr>
+          </thead>
+          <tbody>
+            <% groups.forEach(function(row, iRow) { %>
+            <tr>
+              <td><%= row.name %></td>
+              <td class="text-center"><%= row.size %></td>
+              <td class="text-center" style ="word-break:break-all;"><small><%= row.uid_list %></small></td>
+              <td class="text-center">
+                <div class="dropdown">
+                  <button type="button" 
+                          class="btn btn-xs dropdown-toggle" 
+                          data-toggle="dropdown" 
+                          aria-haspopup="true"
+                          aria-expanded="false">
+                    Action <span class="caret"></span>
+                  </button>
+                  <div id="row<%= iRow %>PopoverAdd" 
+                        tabindex="0" 
+                        data-toggle="popover" 
+                        data-trigger="manual"
+                        data-container="body" 
+                        data-html="true" 
+                        data-placement="auto" 
+                        title="Add Members"
+                        data-content="<%= include('formAddMembers.ejs', {row: row, iRow: iRow }); %>">
+                  </div>
+                  <div id="row<%= iRow %>Popoverdeletemember" 
+                        tabindex="0" 
+                        data-toggle="popover" 
+                        data-trigger="manual"
+                        data-container="body" 
+                        data-html="true" 
+                        data-placement="auto" 
+                        title="Remove Members"
+                        data-content="<%= include('formRemoveMembers.ejs', {row: row, iRow: iRow }); %>">
```

Comment:
It would be nice to improve user deletion (and maybe add) so that the instructor doesn't have to copy/paste the username of the student. For example, there could be a dropdown next to each student with options to remove them from the group, or shift to a different group?